packages/react-admin/src/mui/layout/MenuItemLink.js

```diff
@@ -20,6 +24,7 @@ export class MenuItemLink extends Component {
 
     render() {
         const {
+            activeClassName,
```

Comment:
why do you add that prop? Developers can already pass their own `classes`

Reply:
because it's easier ? I'll remove it if you care

Reply:
yes please